<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>CM</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cm_combobox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Cavity</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="cav_spinbox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Search Start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="search_start_spinbox">
       <property name="minimum">
        <number>500</number>
       </property>
       <property name="maximum">
        <number>15000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Drive Limits</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="PyDMByteIndicator" name="phase_high_byte">
          <property name="toolTip">
           <string/>
          </property>
          <property name="onColor" stdset="0">
           <color>
            <red>252</red>
            <green>33</green>
            <blue>37</blue>
           </color>
          </property>
          <property name="offColor" stdset="0">
           <color>
            <red>41</red>
            <green>200</green>
            <blue>50</blue>
           </color>
          </property>
          <property name="showLabels" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="PyDMLabel" name="phase_low_label">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="PyDMLabel" name="amp_high_label">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="PyDMLabel" name="phase_high_label">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="PyDMByteIndicator" name="amp_high_byte">
          <property name="toolTip">
           <string/>
          </property>
          <property name="onColor" stdset="0">
           <color>
            <red>252</red>
            <green>33</green>
            <blue>37</blue>
           </color>
          </property>
          <property name="offColor" stdset="0">
           <color>
            <red>41</red>
            <green>200</green>
            <blue>50</blue>
           </color>
          </property>
          <property name="showLabels" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="PyDMLabel" name="amp_low_label">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Amp Low</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Amp High</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Phase High</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="PyDMByteIndicator" name="amp_low_byte">
          <property name="toolTip">
           <string/>
          </property>
          <property name="onColor" stdset="0">
           <color>
            <red>252</red>
            <green>33</green>
            <blue>37</blue>
           </color>
          </property>
          <property name="offColor" stdset="0">
           <color>
            <red>41</red>
            <green>200</green>
            <blue>50</blue>
           </color>
          </property>
          <property name="showLabels" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Phase Low</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="PyDMByteIndicator" name="phase_low_byte">
          <property name="toolTip">
           <string/>
          </property>
          <property name="onColor" stdset="0">
           <color>
            <red>252</red>
            <green>33</green>
            <blue>37</blue>
           </color>
          </property>
          <property name="offColor" stdset="0">
           <color>
            <red>41</red>
            <green>200</green>
            <blue>50</blue>
           </color>
          </property>
          <property name="showLabels" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Gains</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <widget class="PyDMByteIndicator" name="PyDMByteIndicator_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="showLabels" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="PyDMSpinbox" name="amp_gain_p_spinbox">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Amplitude</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Phase</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="PyDMSpinbox" name="phase_gain_p_spinbox">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="PyDMByteIndicator" name="PyDMByteIndicator_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="showLabels" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Proportional Gain</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Integral Gain</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="PyDMSpinbox" name="amp_gain_i_spinbox">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="PyDMSpinbox" name="phase_gain_i_spinbox">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPushButton" name="optimize_button">
     <property name="text">
      <string>Optimize</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Ready to Optimize</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMByteIndicator</class>
   <extends>QWidget</extends>
   <header>pydm.widgets.byte</header>
  </customwidget>
  <customwidget>
   <class>PyDMSpinbox</class>
   <extends>QDoubleSpinBox</extends>
   <header>pydm.widgets.spinbox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
